tests/epyccel/test_arrays.py

```diff
@@ -1458,9 +1453,7 @@ def test_array_real_3d_F_array_initialization_2(language):
     assert np.array_equal(x1, x2)
 
 @pytest.mark.parametrize( 'language', [
-        pytest.param("c", marks = [
-            pytest.mark.skip(reason="array function doesn't handle list of variables. See #752"),
-            pytest.mark.c]),
+        pytest.param("c", marks = pytest.mark.c),
         pytest.param("fortran", marks = pytest.mark.fortran),
         pytest.param("python", marks = pytest.mark.python)
     ]
```

Comment:
This can be removed if all languages work. The test will use the global feature to determine the language

Reply:
https://github.com/pyccel/pyccel/pull/1128/commits/8591acc9a9119be6876d5c5fdeb11f14abd6320a